generic_throttle/package.xml

```diff
@@ -0,0 +1,15 @@
+<package format="2">
+  <name>generic_throttle</name>
+  <version>0.1.5</version>
+  <description>This package provides a throttle for ROS topics</description>
+
+  <license>LGPL</license>
+  
+  <maintainer email="mattia.racca@aalto.fi">Mattia Racca</maintainer>
```

Comment:
I guess we should add someone from IPA as a maintainer, too, as it lives in the ipa320 organization...